extensions/amazon-lambda/common-deployment/src/main/java/io/quarkus/amazon/lambda/deployment/LambdaUtil.java

```diff
@@ -76,7 +90,7 @@ public static void generateScripts(String handler, OutputTargetBuildItem target)
         output = copyResource("lambda/manage.sh")
                 .replace("${handler}", handler)
                 .replace("${lambdaName}", lambdaName);
-        writeFile(target, "manage.sh", output);
+        writeExecutableFile(target, "manage.sh", output);
```

Comment:
Should the same be done for `bootstrap-example.sh` which gets created a few lines above this?

Reply:
done